File: Ejercicio 17/ej17.js
let num = 20;
f = "Fizz";
b = "Buzz";

for(let i = 1; i <= num; i++){
    if(i%3 == 0 && i%5 == 0){
        console.log(f+b);
    }else if(i%3 == 0){
        console.log(f);
    }else if(i%5 == 0){
        console.log(b);
    }else{
        console.log(i);
    }
}

File: EjerciciosJava/src/ej17.java
public class ej17 {
    public static void main(String[] args) {
        int num = 20;
        String f = "Fizz";
        String b = "Buzz";

        for (int i = 0; i <= num; i++) {
            if(i%3 == 0 && i%5 == 0){
                System.out.println(f+b);
            }else if(i%3 == 0){
                System.out.println(f);
            }else if(i%5 == 0){
                System.out.println(f);
            }else{
                System.out.println(i);
            }
        }
    }
}
